.github/workflows/sync-tools_and_sync-workflows.yml

```diff
@@ -23,7 +23,22 @@ on:
 
 jobs:
   sync_tools_and_workflows_and_create_catalogue:
-    if: ${{ ( github.event.review.state == 'approved' && github.event.pull_request.draft == false ) || github.event_name == 'workflow_dispatch' }}
+    # First check
+    # 1a. It's a PR that's in an approved state
+    # AND
+    # 1b. It's not a PR to 'Add in images for release'
+    # 2. OR it's been run manually by using the workflow_dispatch command
+    if: |
+      ${{ 
+        (
+          ( 
+            github.event.review.state == 'approved' && 
+            github.event.pull_request.draft == false 
+          ) &&
+          ! startsWith( github.event.pull_request.title, "Add in images for release" ) 
```

Comment:
So we'll need to use this specific PR title for image PRs?

Reply:
Image PRs will be auto-generated through the GitHub actions https://github.com/umccr/cwl-ica/blob/main/.github/workflows/create_workflow_release_asset.yml. PR is created and named by the GitHub actions bot

See https://github.com/umccr/cwl-ica/pull/251 as an example of the PR.